src/v3/src/transformer/identify/transformIdentify.ts

```diff
@@ -53,24 +59,14 @@ export const transformIdentify: IdxStepTransformer = ({ formBag, widgetProps, tr
     options: { content: loc('primaryauth.title', 'login') },
   };
 
-  const identifierElement = getUIElementWithName(
-    'identifier',
-    uischema.elements as UISchemaElement[],
-  ) as FieldElement;
-
   // add username/identifier from config if provided
   if (identifierElement) {
+    // add username/identifier from config if provided
     if (username) {
-      identifierElement.options = {
-        ...identifierElement.options,
-        defaultOption: username,
-      };
+      data.identifier = username;
```

Comment:
Does this solve https://oktainc.atlassian.net/browse/OKTA-521302 ?

Reply:
Yessir it does! I will pull it in to my lane

Reply:
Do you mind to fix that issue in a separate PR, it would be great to fix bug with a test (integration level).

Reply:
Created a PR with tests: https://github.com/okta/okta-signin-widget/pull/2699